src/main/java/hudson/maven/MavenEmbedderUtils.java

```diff
@@ -43,7 +43,6 @@
 import org.codehaus.plexus.classworlds.ClassWorld;
 import org.codehaus.plexus.classworlds.realm.ClassRealm;
 import org.codehaus.plexus.util.IOUtil;
-import org.eclipse.sisu.BeanScanning;
```

Comment:
Was not used anyway. And it was incompatible in sisu.plexus. Maybe there is some magic behind